ecosystem/python/sdk/aptos_sdk/ed25519.py

```diff
@@ -92,6 +94,42 @@ def serialize(self, serializer: Serializer):
         serializer.to_bytes(self.key.encode())
 
 
+class MultiEd25519PublicKey:
```

Comment:
probably should validate threshold and number of keys

maybe move to another file as well?

Reply:
@davidiw requested range checks and expected failure tests have been added.